### PR TITLE
Adding CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Kuljeet-123**
Fixes issue:- #4

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Welcome to <b>React Pincode</b> Project. This file demonstrates the contributing guidelines that every developer/mentor has to adhere for GirlScript Summer of Code'20. This is required in order to create a unified development environment. Let's start.
+
+## The Git Workflow
+The workflow that we'll be using is <b> Dev - Staging - Production</b>. All the Pull Request (PR's) will be made to <b>develop</b> branch.
```
Comment (plxity) [inline CONTRIBUTING.md]: @Kuljeet-123 Change the branch name to master.

**Comment by Kuljeet-123** [comment]
@yellowwoods12  it's been 2 days ...please review my PR and merge it.

**Review by yellowwoods12 — APPROVED** [review]

**Comment by Kuljeet-123** [comment]
@yellowwoods12 Could you please merge it...It's been 11 days.

**Review by plxity — CHANGES_REQUESTED** [review]
Please make this file same as in React-Pincode. We have to maintain the uniformity across all about projects.

**Comment by Kuljeet-123** [comment]
@plxity I have update the changes you requested.Please review it.

**Comment by plxity** [comment]
> Please make this file same as in React-Pincode. We have to maintain the uniformity across all about projects.

@Kuljeet-123 please make the changes.

**Comment by Kuljeet-123** [comment]
@plxity I have update the changes you requested.Please review it.

**Review by yellowwoods12 — APPROVED** [review]

**Review by plxity — APPROVED** [review]

**Pull request merged**